client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -177,6 +204,20 @@ public ReaderGroupConfigBuilder startFromCheckpoint(final Checkpoint checkpoint)
            return this;
        }
 
+       /**
+        * Set the retention config for the {@link ReaderGroup}.
+        * For Consumption based retention of data in the Stream, this field should be set to
```

Comment:
"should be set to" is not helpful. 
Specify what happens with each setting including NO_IMPACT

Reply:
I have addressed that in the javadoc for the enum.